src/app/tabs/run-tab.js

```diff
@@ -797,21 +788,11 @@ function settings (container, appAPI, appEvents, opts) {
       }
     }
 
-    /*
-    addIfNotExists({
-      name: 'KEVM Testnet',
-      context: 'custom-rpc-kevm-testnet',
-      chainId: undefined,
-      rpcUrl: 'https://kevm-testnet.iohkdev.io:8546/',
-      vm: 'evm'
-    })
-    */
-
     addIfNotExists({
       name: 'IELE Testnet',
       context: 'custom-rpc-iele-testnet',
       chainId: undefined,
-      rpcUrl: 'https://staging.iele.dev-mantis.iohkdev.io:8546/',
+      rpcUrl: window.location.protocol + `//` + window.location.hostname + ':8546/',
```

Comment:
This will not work if we develop Remix IDE locally with hostname like `127.0.0.1` or `localhost` where there is no node running..

Reply:
Actually, I think this should be absolute URL as well.